File: app/src/main/java/com/peluac/vataret/ucatale/model/RecordInfo.java
package com.peluac.vataret.ucatale.model;

import android.graphics.Bitmap;

import java.io.Serializable;

/**
 * create by ping at 2023/9/12
 */
public class RecordInfo implements Serializable {

    private Bitmap coupleBitmap;
    private String coupleBitmapUri;
    private String time;

    public String getCoupleBitmapUri() {
        return coupleBitmapUri;
    }

    public void setCoupleBitmapUri(String coupleBitmapUri) {
        this.coupleBitmapUri = coupleBitmapUri;
    }


    public Bitmap getCoupleBitmap() {
        return coupleBitmap;
    }

    public void setCoupleBitmap(Bitmap coupleBitmap) {
        this.coupleBitmap = coupleBitmap;
    }


    public RecordInfo() {
    }


    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }
}

File: app/src/main/java/com/gwesupera/goldbole/model/RecordInfo.kt
package com.gwesupera.goldbole.model

import android.graphics.Bitmap
import java.io.Serializable

/**
 * create by ping at 2023/9/12
 */
class RecordInfo : Serializable {
    var coupleBitmap: Bitmap? = null
    var coupleBitmapUri: String? = null
    var time: String? = null
}